Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt

```diff
@@ -180,9 +183,25 @@ class HomeFragmentViewModel @AssistedInject constructor(
         mutableListOf<HomeItem>().apply {
             when {
                 statePCR is SubmissionStatePCR.TestPositive || statePCR is SubmissionStatePCR.SubmissionDone -> {
-                    // Don't show risk card
+                    if (testPCR != null && isOlderThanThreshold(
```

Comment:
here we are using `when` expression if one case meets the condition the other won’t run. what happens if we have two positive tests one is older and the other is younger, will the card be shown or hidden?